.travis.yml

```diff
@@ -98,7 +99,7 @@ notifications:
 
 ## list of services to be running
 services:
-  - postgresql
+  - docker
```

Comment:
do we need docker even if we use `sudo: required`? Would it be enough to install libgdal-dev?

Reply:
I'm not sure I follow -- we install libgdal-dev already (see my comment below), the problem is that there's no version of it that's acceptable to both postgis and rgdal. Are you suggesting that we install both gdal 1 and gdal 2 alongside each other?